dist/dss.js

```diff
@@ -1,53 +1,56 @@
 //File : src/dss.js
 
-
-(function(world){
+(function(world) {
   'use strict';
 
   /**
    * [dss description]
    * @return {[Object]}
    */
-  world.dss = function(){ 
-
+  world.dss = function() {
     return {
       core : {}
     };
-
   }();
-
 })(this);
+
 //File : src/core/dss.core.ajax.js
 
-(function(dss){
+(function(dss) {
 	'use strict';
 
- dss.core.ajax = function(args){
- 	var callback = args.callback || function(){return false;};
+ dss.core.ajax = function(args) {
+ 	var callback = args.callback || function() { return false; };
  	var url = args.url;
  	var method = args.method || 'GET';
  	var data = args.data || false;
-
  	var xhr;
- 	
- 	if(typeof XMLHttpRequest !== 'undefined') xhr = new XMLHttpRequest();
- 	else {
- 		var versions = ["MSXML2.XmlHttp.5.0", 
- 		"MSXML2.XmlHttp.4.0",
- 		"MSXML2.XmlHttp.3.0", 
- 		"MSXML2.XmlHttp.2.0",
- 		"Microsoft.XmlHttp"];
- 		
+
+ 	if (typeof XMLHttpRequest !== 'undefined') {
+    xhr = new XMLHttpRequest();
+  } else {
+ 		var versions = [
+      'MSXML2.XmlHttp.5.0',
```

Comment:
Yes, but is better to see the problem at your code editor, using tabs itself is not good.. since tabs doesn't have a fixed size, but the worse is that your code has tabs and space mixed, check it out:

<img width="335" alt="captura de tela 2015-09-22 as 19 50 59" src="https://cloud.githubusercontent.com/assets/2164551/10033994/71ca46f4-6163-11e5-9522-48e7bdd0b796.png">